### DISCUSSION TITLE
Profiler variables object depth

### DISCUSSION THREAD

**Discussion by costeaalex** [symfony/symfony#42795] [Q&A]
I am browsing in the profiler panel and looking at a Session variable that contains other objects which seem not to be listed but appear as 
![image](https://user-images.githubusercontent.com/11831652/131351387-facb0326-94ab-4ec6-ab46-f38e1a115bba.png)
and I was wondering if there is a way to expand and view these objects within the object. I could not find anything relevant in the documentation and it doesn't seem to be affected by Xdebug depth setting.

**Comment by michaoj** [comment]
Did you try to set the minimal depth ?

https://symfony.com/doc/current/reference/configuration/debug.html#min-depth

**Comment by costeaalex** [comment]
Nope, this is something related to the web_profiler not the var-dumper or debug-bundle. I did give it a shot to install these two and add this configuration but no joy. Thanks for the help.

**Reply by stof** [reply]
The web profiler uses the VarDumper component to dump the values.

**Reply by costeaalex** [reply]
It does use the var dumper because it installs it through composer and I see it in the lock file but the setting mentioned above pertains to the debug-bundle and I have installed explicitly (in composer.json) both var-dumper and debug-bundle, added the min-depth setting in my config and verified that it is taken into account but the behavior is the same in that the objects are logged one level deep. I can provide a more detailed example if you wish and think you can help me out. Thanks

**Reply by stof** [reply]
check whether this object with id 215 is already dumped earlier in the output. The VarDumper avoids dumping details multiple times.

**Reply by costeaalex** [reply]
What I am looking at is a session variable and I have checked that is is not at all outputted anywhere else in the dashboard of the profiler.

You can see in this image below that all of the objects are just flat and there's no way to go in depth:

My next course of action will probably be a fresh Symfony install where I check this in depth :).

![image](https://user-images.githubusercontent.com/11831652/131878653-e448824f-427d-456a-a351-69b4a7975249.png)
